rest-server/test/user.js

```diff
@@ -0,0 +1,257 @@
+require('babel-register');
+require('babel-polyfill');
+
+const { User } = require('../src/config/database/collections/userCollections');
+const chai = require('chai');
+const chaiHttp = require('chai-http');
+const server = require('../src/');
+const should = chai.should();
+const expect = chai.expect;
+
+chai.use(chaiHttp);
+
+describe('Users', () => {
+  beforeEach( done => {
+    User.remove({}, err => {
+      done();
+    });
+  });
+
+  /*
+  * /GET
+  */
+  describe('/GET user', () => {
+    it('should list ALL users on /users GET', done => {
+      chai.request(server)
+        .get('/api/users')
+        .end( (err, res) => {
+          res.should.have.status(200);
+          res.should.be.json;
+          expect(res.body).to.be.an('array');
+          res.body.length.should.equal(0);
+          done();
+        });
+    });
+    
+    it('should list a SINGLE user on /users/:id GET', done => {
+      const newUser = new User({
+        name: 'Test User',
+        dob: '08/27/2018',
+        address: 'Los Angeles',
+        description: 'This is a test'
+      });
+      newUser.save( (err, data) => {
+        chai.request(server)
+          .get(`/api/users/${data.id}`)
+          .end( (err, res) => {
```

Comment:
Chai is pretty cool because it has some plugins to extend base functionality. https://github.com/chaijs/deep-eql allows you to compare a complete structure without breaking each node out for an assertion.